### PR TITLE
Add distribution module for jacoco report aggregation

### PR DESCRIPTION
**Pull request by pl-buiquang**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
https://github.com/powsybl/powsybl-afs/issues/6

**Other information:**
Cassandra tests are uncommented (second point of the issue), and fixes for broken tests added

### CODE REVIEW

**Comment by geofjamg** [comment]
Looking at Sonar report, it seems that code coverage of ws part is not fully taken into account in particulat client part is 0%. Any idea what could be the problem?

**Comment by pl-buiquang** [comment]
> Looking at Sonar report, it seems that code coverage of ws part is not fully taken into account in particulat client part is 0%. Any idea what could be the problem?

No, actually, I was going to comment that I'm not sure if what I did is the right (and working) way to do this..

**Comment by pl-buiquang** [comment]
Maybe this will be taken into account when merged ?

File: pom.xml [hunk 1]
```diff
@@ -64,6 +64,7 @@
         <module>afs-security-analysis-local</module>
         <module>afs-storage-api</module>
         <module>afs-ws</module>
+        <module>distribution-afs</module>
```
Comment (mathbagu) [inline pom.xml]: is this module name Ok or should it be afs-distribution?

File: distribution-afs/pom.xml [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+    </profiles>
+
+</project>
```
Comment (mathbagu) [inline distribution-afs/pom.xml]: Add an empty line at the end of the file

File: distribution-afs/pom.xml [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+            <artifactId>powsybl-afs-ws-server-utils</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.powsybl</groupId>
+            <artifactId>powsybl-afs-ws-storage</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.powsybl</groupId>
```
Comment (mathbagu) [inline distribution-afs/pom.xml]: Replace all com.powsybl occurrences by ${project.groupId}

**Comment by mathbagu** [comment]
@jonenst is this correction ok for you, as our maven expect. I remember you did something in powsybl-core recently to make jacoco working well in command line?

**Comment by jonenst** [comment]
sonar.converage.jacoco.report.xmlReportPaths was missing to direct all modules to the aggregated report

**Comment by geofjamg** [comment]
> sonar.converage.jacoco.report.xmlReportPaths was missing to direct all modules to the aggregated report

Ok thanks.
77% coverage after merge, I think we can merge it as it is.

**Pull request merged**